lib/node_modules/@stdlib/math/base/special/sincospi/README.md

```diff
@@ -0,0 +1,78 @@
+# sincospi
+
+> Simultaneously compute the [sine][@stdlib/math/base/special/sin] and [cosine][@stdlib/math/base/special/cos] of a number times [π][@stdlib/math/constants/pi].
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var sincospi = require( '@stdlib/math/base/special/sincospi' );
+```
+
+#### sincospi( \[out,] x )
+
+Simultaneously computes the [sine][@stdlib/math/base/special/sin] and [cosine][@stdlib/math/base/special/cos] of a `number` times [π][@stdlib/math/constants/pi] more accurately than `sincos(pi*x)`, especially for large `x`..
+
+```javascript
+var v = sincospi( 0.0 );
+// returns [ ~0.0, ~1.0 ]
+
+v = sincospi( 0.5 );
+// returns [ ~1.0, ~0.0 ]
+
+v = sincospi( 0.1 );
+// returns [ ~0.309, ~0.951 ]
+
+v = sincospi( NaN );
+// returns [ NaN, NaN ]
+```
+
+By default, the function returns the sine and cosine as a two-element `array`. To avoid extra memory allocation, the function supports providing an output (destination) object.
```

Comment:
Maybe "returns the result as a two-element `array`"? But could also leave as-is as it should be clear from context that everything is scaled by pi?